src/application/contribution-case/contribution-case.routes.js

```diff
@@ -0,0 +1,71 @@
+import { Router } from "express";
+import { message } from "../../utils/message.js";
+import { param } from "express-validator";
+import {
+  createContribution,
+  deleteContribution,
+  getContributions,
+} from "./contribution-case.controller.js";
+import { pagination } from "../../middleware/pagination.js";
+import {
+  locationSearchQueryParams,
+  locationValidation,
+} from "../../middleware/coordinates-validation.js";
+import { publicCaseValidation } from "../../middleware/public-case-validation.js";
+import { validateChecks } from "../../middleware/validate-checks.js";
+import { custom } from "../../middleware/custom.js";
+import { Contribution } from "./contribution-case.model.js";
+import { ContributionCaseNotFoundError } from "./contribution-case.errors.js";
+import {
+  filepathsValidation,
+  validateOptionalsFilepathsAreInStaleContent,
+} from "../../middleware/filepaths.js";
+
+const router = Router();
+
+router
+  .route("/")
+  .get(
+    [...pagination, ...locationSearchQueryParams, validateChecks],
```

Comment:
este middleware esta mal. no deberia tener `locationSearchQueryParams` este middleware es solo para `public-case` y `anonympus-case`

```suggestion
    [...pagination, validateChecks],
```